tests/test_core_to_contrib.py

```diff
@@ -16,787 +16,814 @@
+    ),
+    (
+        "airflow.gcp.operators.gcs.GoogleCloudStorageCreateBucketOperator",
+        "airflow.contrib.operators.gcs_operator.GoogleCloudStorageCreateBucketOperator",
+    ),
+]
+SENSOR = [
+    (
+        "airflow.gcp.sensors.bigtable.BigtableTableWaitForReplicationSensor",
+        "airflow.contrib.operators.gcp_bigtable_operator."
+        "BigtableTableWaitForReplicationSensor",
+    ),
+    (
+        "airflow.gcp.sensors.cloud_storage_transfer_service."
+        "GCPTransferServiceWaitForJobStatusSensor",
+        "airflow.contrib.sensors.gcp_transfer_sensor."
+        "GCPTransferServiceWaitForJobStatusSensor",
+    ),
+    (
+        "airflow.gcp.sensors.pubsub.PubSubPullSensor",
+        "airflow.contrib.sensors.pubsub_sensor.PubSubPullSensor",
+    ),
+    (
+        "airflow.gcp.sensors.bigquery.BigQueryTableSensor",
+        "airflow.contrib.sensors.bigquery_sensor.BigQueryTableSensor",
+    ),
+    (
+        "airflow.gcp.sensors.gcs.GoogleCloudStorageObjectSensor",
+        "airflow.contrib.sensors.gcs_sensor.GoogleCloudStorageObjectSensor",
+    ),
+    (
+        "airflow.gcp.sensors.gcs.GoogleCloudStorageObjectUpdatedSensor",
+        "airflow.contrib.sensors.gcs_sensor.GoogleCloudStorageObjectUpdatedSensor",
+    ),
+    (
+        "airflow.gcp.sensors.gcs.GoogleCloudStoragePrefixSensor",
+        "airflow.contrib.sensors.gcs_sensor.GoogleCloudStoragePrefixSensor",
+    ),
+    (
+        "airflow.gcp.sensors.gcs.GoogleCloudStorageUploadSessionCompleteSensor",
+        "airflow.contrib.sensors.gcs_sensor.GoogleCloudStorageUploadSessionCompleteSensor",
+    ),
+]
+
+ALL = HOOKS + OPERATOR + SENSOR
+
 
 class TestMovingCoreToContrib(TestCase):
+    @staticmethod
+    def assert_warning(msg: str, warning: Any):
+        error = "Text '{}' not in warnings".format(msg)
+        assert any(msg in str(w) for w in warning.warnings), error
+
     def assert_is_subclass(self, clazz, other):
-        self.assertTrue(issubclass(clazz, other))
+        self.assertTrue(issubclass(clazz, other), "{} is not subclass of {}".format(clazz, other), )
 
-    def assert_proper_import(self, new_resource: str, old_resource: str):
-        old_path, _, old_class_name = old_resource.rpartition(".")
-        new_path, _, new_class_name = new_resource.rpartition(".")
-        with self.assertWarns(DeprecationWarning):
+    def assert_proper_import(self, old_resource, new_resource):
+        new_path, _, _ = new_resource.rpartition(".")
+        old_path, _, _ = old_resource.rpartition(".")
+        with self.assertWarns(DeprecationWarning) as warning_msg:
             # Reload to see deprecation warning each time
-            old_module = importlib.reload(importlib.import_module(old_path))
-        new_module = importlib.import_module(new_path)
+            importlib.reload(importlib.import_module(old_path))
+            self.assert_warning(new_path, warning_msg)
```

Comment:
I like that we assert proper warning message here !

Reply:
:)